nucleus/test-utils/src/main/java/org/glassfish/tests/utils/ServerUtils.java

```diff
@@ -40,22 +44,57 @@
  */
 public class ServerUtils {
 
+    /**
+     * Tries to allocate a free local ports, avoid duplicates and excluded numbers.
+     *
+     * @param count count of free ports to find.
+     * @param excluded list of ports to avoid
+     * @return a modifiable queue of free local port numbers.
+     * @throws IllegalStateException if it fails for 20 times
+     */
+    public static Queue<Integer> getFreePorts(int count, int... excluded) throws IllegalStateException {
```

Comment:
Out of curiosity - can see only call for `getFreePorts(int)`. Where is the vararg option used?

What is the `excluded` param used for?